subxt/src/backend/unstable/follow_stream_driver.rs

```diff
@@ -380,6 +380,107 @@ struct SubscriberDetails<Hash: BlockHash> {
     waker: Option<Waker>,
 }
 
+/// A stream that subscribes to finalized blocks
+/// and indicates whether a block was missed if was restarted.
+#[derive(Debug)]
+pub struct FollowStreamFinalizedHeads<Hash: BlockHash, F> {
+    stream: FollowStreamDriverSubscription<Hash>,
```

Comment:
This type @jsdw :point_up:

Reply:
that struct is `pub` in this file though, so we could import it from a separate file OK I think? :D

Reply:
it's a pita because the test helpers are not accessible outside this module :(

Reply:
Ah I see! either way it's all good and we can always shuffle stuff about in the future or whatever anyways!